expr/boolean.go

```diff
@@ -201,7 +202,7 @@ var compareString = map[string]func(string, string) bool{
 	"<=": func(a, b string) bool { return a <= b },
 }
 
-func CompareBstring(op string, pattern zng.Bstring) (Boolean, error) {
+func CompareBstring(op string, pattern bstring) (Boolean, error) {
```

Comment:
I didn't notice `Bstring` was part of a public interface when I suggested changing it to `bstring`. :facepalm:

I'm happy to let that ride until we rework `parseLiteral`, though.